app-misc/jigit/jigit-1.20.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="A tool to ease the distribution of very large files"
+DESCRIPTION="Jigsaw Download, or jigdo, is a tool designed to ease the \
+distribution of very large files over the internet such as CD or DVD \
+images. Its aim is to make downloading these images as easy as a click \
+on a direct download link in a browser, while avoiding all the problems \
+that server administrators could have with hosting large files."
+HOMEPAGE="https://www.einval.com/~steve/software/JTE/"
+COPYRIGHT="2004-2014 Steve McIntyre
+	2000-2010 George Danchev, Thomas Schmitt"
+LICENSE="GNU GPL v2
+	GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/download/jigit_$portVersion.orig.tar.gz"
+CHECKSUM_SHA256="459285bd743645381912301578316f2bfafb99ce1c10140a01af2b6cb011f193"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	jigit$secondaryArchSuffix = $portVersion
+	cmd:extract_data$commandSuffix = $portVersion
+	cmd:jigdump$commandSuffix = $portVersion
+	cmd:jigit$commandSuffix = $portVersion
+	cmd:jigsum$commandSuffix = $portVersion
+	cmd:jigit_mkimage$commandSuffix = $portVersion
+	cmd:mkjigsnap$commandSuffix = $portVersion
+	cmd:rsyncsum$commandSuffix = $portVersion
+	lib:libjte$secondaryArchSuffix = 1.0.0 compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libjte${secondaryArchSuffix}_devel = $portVersion
+	devel:libjte$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libjte$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config
```

Comment:
Doesn't hurt if you add it if you are building for secondaryarch :)

Reply:
@Begasus @fbrosson So I guess I should add `$secondaryArch`?

Reply:
I'd say yes, because we do that on all other recipes, but I'm ok without it, because it looks like it isn't needed here.

Yeah, adding it seems slightly better.